task_schedule3.pl

```diff
@@ -322,7 +322,8 @@ sub check_outputs {
 ##***************************************************************************
     my $cronjob = shift;
 
-    my $watch_config = io(File::Temp::tmpnam);
```

Comment:
Looks like the fix I had to use File::Temp::tmpnam instead of POSIX::tmpnam isn't working because File::Temp::tmpnam is context sensitive and I think it is defaulting to list context when called in io().